Content.Shared/Humanoid/Markings/MarkingManager.cs

```diff
@@ -67,7 +67,7 @@ public IReadOnlyDictionary<string, MarkingPrototype> MarkingsByCategoryAndSpecie
 
             foreach (var (key, marking) in MarkingsByCategory(category))
             {
-                if ((markingPoints.OnlyWhitelisted || markingPoints.Points[category].OnlyWhitelisted) && marking.SpeciesRestrictions == null)
+                if ((markingPoints.OnlyWhitelisted || !markingPoints.Points.TryGetValue(category, out var val) || val.OnlyWhitelisted) && marking.SpeciesRestrictions == null)
```

Comment:
Needs L5 comment, otherwise good to go unless we want to remove restrictions entirely.